torch/distributed/_tensor/ops/tensor_ops.py

```diff
@@ -59,6 +57,45 @@ def default_strategy(mesh: DeviceMesh, op_schema: OpSchema) -> StrategyType:
     )
 
 
+@register_op_strategy(
+    [
+        aten.equal.default,
+        aten.is_same_size.default,
```

Comment:
yep is_same_size would never hit. I mainly leave it here because if logically these two share the same logic if we don't do any dispatching optimizations in this PR, and for completeness I was mostly thinking if we plan to explore "single graph expansion like dist compiler" in the future again, having all ops registered to one table might be better, but I can remove it if you feel we want to keep things consistent ;)

Because for `is_same_size` we don't need to run propagation as we can just directly compare the DTensor shapes, the dispatching logic is more like a optimization.

Reply:
ok that convinced me, this is fine

Reply:
Do you think it's a good idea to keep all lists of ops (e.g. `skip_prop_list`, `random_op_list`) in one file for better readability?